### PR TITLE
Media lib upgrade Buffet.js version

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:
This PR also fixes the date format of the filters

### CODE REVIEW

File: packages/strapi-plugin-upload/admin/src/utils/createNewFilesToDownloadArray.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const createNewFilesToDownloadArray = (filesURLArray, alreadyUploadedFiles) => {
       fileInfo: {
         alternativeText: '',
         caption: '',
-        name: current,
+        name: current.substring(current.lastIndexOf('/') + 1),
```
Comment (alexandrebodin) [inline packages/strapi-plugin-upload/admin/src/utils/createNewFilesToDownloadArray.js:32]: I think this won't be enough here (we need to handle query strings :/)

We should use URL to parse the url get the name:

```js
try {
        const url = new URL(current);

        const name =  decodeURIComponent(url.pathname.substring(url.pathname.lastIndexOf('/') + 1))

 } catch (err) {
  // invalid url
} 
```

Comment (soupette) [reply packages/strapi-plugin-upload/admin/src/utils/createNewFilesToDownloadArray.js:32]: Ok I'll make an update then

File: packages/strapi-plugin-upload/admin/src/containers/InputModalStepperProvider/index.js [hunk 2]
```diff
@@ -352,11 +352,19 @@ const InputModalStepperProvider = ({
     });
 
     const requests = filesToUpload.map(
-      async ({ file, fileInfo, originalIndex, abortController }) => {
+      async ({ file, fileInfo, originalIndex, originalName, abortController }) => {
         const formData = new FormData();
         const headers = {};
+        const infos = clone(fileInfo);
+
+        if (originalName === infos.name) {
```
Comment (soupette) [inline packages/strapi-plugin-upload/admin/src/containers/InputModalStepperProvider/index.js:361]: @alexandrebodin removing the fileInfo.name

File: packages/strapi-plugin-upload/admin/src/containers/ModalStepper/index.js [hunk 3]
```diff
@@ -355,9 +355,14 @@ const ModalStepper = ({
     });
 
     const requests = filesToUpload.map(
-      async ({ file, fileInfo, originalIndex, abortController }) => {
+      async ({ file, fileInfo, originalName, originalIndex, abortController }) => {
         const formData = new FormData();
         const headers = {};
+
+        if (originalName === fileInfo.name) {
```
Comment (soupette) [inline packages/strapi-plugin-upload/admin/src/containers/ModalStepper/index.js:370]: Same here

**Pull request merged**